modules/data.atmosphere/R/met_temporal_downscale.Gaussian_ensemble.R

```diff
@@ -199,7 +290,57 @@ met_temporal_downscale.Gaussian_ensemble <- function(in.path, in.prefix, outfold
           }
           dwnsc_day <- vector()
           for (n in seq_len(div)) {
-            dwnsc_day[n] <- stats::rnorm(1, mean = sour[x], sd = stats::sd(a[lowday:highday]))
+            if (u == "volume_fraction_of_condensed_water_in_soil") {
+              idx <- (x - 1) * div + n
+              current_precip <- ifelse(idx <= length(precip) && idx >= 1, 
+                                       precip[idx], 0)
+              if (is.na(current_precip)) current_precip <- 0
+              antecedent_sm <- ifelse(idx > 1, 
+                                      sour[max(1, x-1)], 
+                                      sour[x])
+              base_sd <- stats::sd(a[lowday:highday], na.rm = TRUE)
+              sm_cv <- stats::sd(sour, na.rm = TRUE) / mean(sour, na.rm = TRUE)
+              if (!is.na(sm_cv) && is.finite(sm_cv)) {
+                sm_fc <- stats::quantile(sour, 0.75, na.rm = TRUE)
+                moisture_stress <- abs(antecedent_sm - sm_fc) / sm_fc
+                uncertainty_factor <- 1.0 + sm_cv * moisture_stress
+                if (current_precip > 0) {
+                  uncertainty_factor <- uncertainty_factor * 1.2  # 20% increase during precipitation
+                }
+              } else {
+                uncertainty_factor <- 1.0
+              }
+              uncertainty_factor <- pmax(0.7, pmin(uncertainty_factor, 1.8))
+              sd_adj <- base_sd * uncertainty_factor
+              soil_moisture <- stats::rnorm(1, mean = sour[x], sd = sd_adj)
+              dwnsc_day[n] <- pmax(0, pmin(soil_moisture, 1.0))
+            } else if (u == "relative_humidity") {
+              base_sd <- stats::sd(a[lowday:highday], na.rm = TRUE)
+              if (is.na(base_sd) || base_sd <= 0) {
+                base_sd <- 5.0  
+              }
+              temp_idx <- (x - 1) * div + n
+              if (temp_idx > 0 && temp_idx <= length(df$air_temperature) && x <= length(source$air_temperature)) {
+                current_temp_c <- df$air_temperature[temp_idx] - 273.15
+                source_temp_c <- source$air_temperature[x] - 273.15
+                
+                if (current_temp_c > -40 && current_temp_c < 50 && source_temp_c > -40 && source_temp_c < 50) {
+                  # magnus formula for saturation vapor pressure (kPa)
+                  es_current <- 0.61078 * exp((17.27 * current_temp_c) / (current_temp_c + 237.3))
+                  es_source <- 0.61078 * exp((17.27 * source_temp_c) / (source_temp_c + 237.3))
```

Comment:
```suggestion
                  # convert temp to saturation vapor pressure (es, kPa)
                  es_current <- PEcAn.data.atmosphere::t2es(current_temp_c)
                  es_source <- PEcAn.data.atmosphere::t2es(source_temp_c)
```


Using existing functions (updated as of #3597 - which supports Magnus approach but uses ClausiusClapeyron by default) 

```r
  
```

(using default ClausiusClapeyron method because that is the approach used most commonly elsewhere in the database).

It is preferred to use documented functions like PEcAn.atmosphere::t2es and units::ud_convert reduces opportunities for error or makes them easier to find and replace, and also helps document the source of the method.

Reply:
now used `t2es`